wikitemplate-ios.md

```diff
@@ -27,78 +27,78 @@
 
 ## Favourites
 
-- [ ] Verify editing favourite and changing URL updates the favicons accordingly
+- [ ] Verify editing favourites and changing their URLs updates the favicons accordingly
 - [ ] Verify that you can remove favourites
-- [ ] Verify that you can add new favourites from the share menu
+- [ ] Verify that you can add new favourites from the `Share with...` menu
 
 ## Context menus
 
 - [ ] Make sure context menu items in the URL bar work
 - [ ] Make sure context menu items on content work with no selected text
 - [ ] Make sure context menu items on content work with selected text
 - [ ] Make sure context menu items on content work inside an editable control (input, textarea, or contenteditable)
-- [ ] Context menu: verify you can Open in Background Tab, and Open in Private Tab
+- [ ] Context menu: verify you can `Open in New Tab` and `Open in Private Tab`
```

Comment:
This is not the same. Open tab in background is a different check from open in new tab. 

This check needs to be changes to `Verify you can links in a new tab or private tab. Ensure you can open links in background tabs.`

Reply:
Thanks, will fix.